Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -168,39 +197,18 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
         cell.connectingForwards = (indexPath.item != 0)
         cell.isPrivate = isPrivate
         
+        cell.setNeedsDisplay()
```

Comment:
This call is needed because the cells draw the connecting lines in the drawRect but sometimes if the cell is reused it won't redraw the correct line (whether it should connect up, down or both ways).